clients/rust/Cargo.toml

```diff
@@ -22,4 +22,5 @@ thiserror = "^1.0"
 assert_matches = "1.5.0"
 solana-program-test = "^1.16"
 solana-sdk = "^1.16"
+spl-memo = "^4.0.0"
```

Comment:
Is this needed?

Reply:
Nope, left over from when I was trying to perform a transfer with the memo extension enabled. Then I went down the route of changing the owner of the account "manually", which was much easier. I will remove it.